src/screens/About/About.tsx

```diff
@@ -62,57 +83,108 @@ const About = () => {
-                    url={logo.url}
-                    altText={logo.altText}
-                    key={logo.altText}
-                  />
-                ))}
-              </StyledGridContainer>
-            </Fragment>
-          );
-        })}
-        <AboutHeading2 id="who-we-serve">
-          {aboutContent.whoWeServeHeader}
-        </AboutHeading2>
-        <PartnersSectionWrapper>
-          <MarkdownContent source={aboutContent.whoWeServeContentA} />
-          <LogoGrid logos={aboutContent.governmentLogos} />
-          <MarkdownContent source={aboutContent.whoWeServeContentB} />
-        </PartnersSectionWrapper>
+        <HashWrapper id="about">
+          <AboutHeading1>{aboutContent.aboutHeader}</AboutHeading1>
+          <MarkdownContent source={aboutContent.aboutContent} />
+          <ButtonContainer>
+            <Button to="/about#team" cta="Meet our team" />
+            <Button to="/about#contact-us" cta="Contact us" />
+          </ButtonContainer>
+        </HashWrapper>
+        <HashWrapper id="mission">
+          <AboutHeading1>{aboutContent.missionHeader}</AboutHeading1>
+          <SectionContent>
+            <MarkdownContent source={aboutContent.missionContent} />
+            {aboutContent.commitmentsContent.map(
+              (commitment: CommitmentsContent, idx: number) => {
+                return (
+                  <CommitmentItem key={`commitment-section-${idx}`}>
+                    <CommitmentIcon
+                      src={commitment.icon}
+                      alt={commitment.altText}
+                    />
+                    <CommitmentBody>
+                      <MarkdownContent source={commitment.copy} />
+                    </CommitmentBody>
+                  </CommitmentItem>
+                );
+              },
+            )}
+            <ButtonContainer>
+              {/* fixme: find the right urls for this section */}
+              <Button to="/data-api#faq" cta="See our data sources" />
+              <Button to="/data-api#faq" cta="Download our data" />
+            </ButtonContainer>
+          </SectionContent>
+        </HashWrapper>
+        <HashWrapper id="impact">
+          <AboutHeading1>{aboutContent.impactHeader}</AboutHeading1>
+          <SectionContent>
+            <MarkdownContent source={aboutContent.impactContent} />
+          </SectionContent>
+          <ButtonContainer>
+            <Button to="/case-studies" cta="View case studies" />
+          </ButtonContainer>
+        </HashWrapper>
+        <HashWrapper id="partners">
+          <AboutHeading1>{aboutContent.partnersHeader}</AboutHeading1>
+          <SectionContent>
+            <ExpandableContainer
+              collapsedHeightMobile={275}
+              collapsedHeightDesktop={565}
+              tabTextCollapsed={<>More</>}
+              tabTextExpanded={<>Less</>}
+              trackingLabel="Partners"
+              trackingCategory={EventCategory.NONE}
```

Comment:
need correct tracking label/category here

Reply:
can add `ABOUT` to the [EventCategory enum](https://github.com/covid-projections/covid-projections/blob/develop/src/components/Analytics/utils.ts#L35), and "Partners logos" as a tracking label sounds good